content/question/1.md

```diff
@@ -0,0 +1,103 @@
+      beginner in field of robotics as well as in gazebo .plz guide me ,i am attachin
+      my modified code here
+    CreationDate: '2013-07-30T15:49:01Z'
+    OriginalPostID: 4000
+    OwnerDisplayName: suvrat
+    OwnerKarma: 53
+    OwnerProfileUrl: https://answers.gazebosim.org/users/438/suvrat/
+  - BodyMarkdown: "   double target_dist = 2.0;\n\n       if (min_dist < this->laser->GetRangeMax())\n\
+      \      {\n\t\n\t\n        double torque = this->gain *( (min_dist - target_dist)*0.1\
+      \ );\n\t\n\t\n        this->leftWheelJoint->SetForce(0, torque);\n        this->rightWheelJoint->SetForce(0,\
+      \ torque);\n      } \n\n\telse \n{\n\t\nthis->leftWheelJoint->SetForce(0,0.0);\n\
+      \        this->rightWheelJoint->SetForce(0,0.0);\n}\n   "
+    CreationDate: '2013-07-30T15:51:34Z'
+    OriginalPostID: 4003
+    OwnerDisplayName: suvrat
+    OwnerKarma: 53
+    OwnerProfileUrl: https://answers.gazebosim.org/users/438/suvrat/
+  - BodyMarkdown: as i can't paste my complete code here i just paste the part which
+      i had modified .
+    CreationDate: '2013-07-30T15:53:35Z'
+    OriginalPostID: 4004
+    OwnerDisplayName: suvrat
+    OwnerKarma: 53
+    OwnerProfileUrl: https://answers.gazebosim.org/users/438/suvrat/
+  CreationDate: '2012-09-19T23:09:50Z'
+  OriginalPostID: 2
+  OwnerDisplayName: nkoenig
+  OwnerKarma: 7676
+  OwnerProfileUrl: https://answers.gazebosim.org/users/5/nkoenig/
+  UpvoteCount: 0
+- Accepted: false
+  BodyMarkdown: i m begginer in gazebo but i solved the problem using SetDamping().if
+    too obvious its k ,i just tell how i done in my case.
+  Comments: []
+  CreationDate: '2013-08-01T14:13:20Z'
+  OriginalPostID: 4037
+  OwnerDisplayName: suvrat
+  OwnerKarma: 53
+  OwnerProfileUrl: https://answers.gazebosim.org/users/438/suvrat/
+  UpvoteCount: 0
+- Accepted: false
+  BodyMarkdown: "Do you guys install perf using following commands and it works for\
+    \ you?\r\n\r\nInstall perf\r\nsudo apt-get install linux-tools\r\nRecord data\r\
+    \nperf record gzserver worlds/pr2.world\r\nGenerate a report\r\nperf report\r\n\
+    \r\n[Beach Wall Murals](https://www.murals.shop/beach-wall-murals/)\r\n[Wallpaper\
+    \ Murals](https://www.murals.shop)\r\n[Flowers Wall Murals](https://www.murals.shop/flowers-wall-murals/)\r\
+    \n[Wall Murals](https://www.murals.shop/designers-choice-wall-murals/)"
+  Comments: []
+  CreationDate: '2017-09-23T08:32:31Z'
+  OriginalPostID: 17305
+  OwnerDisplayName: adaman05
+  OwnerKarma: 1
+  OwnerProfileUrl: https://answers.gazebosim.org/users/3828/adaman05/
+  UpvoteCount: 0
+Comments:
+- BodyMarkdown: i have the same kind of problem ,i tried to modify the c++ code
+    in tutorial http://gazebosim.org/wiki/Tutorials/1.3/control_robot/mobile_base_laser
+    .it also not stopping.i have setForce to zero even it is not stooping .I am
+    beginner in field of robotics as well as in gazebo .plz guide me ,i am attachin
+    my modified code here
```

Comment:
It's weird that this is unqouted and still seems to work fine. I expect because of the indentation.

Reply:
That's true. I'll update this post in a new dump example version

Reply:
Updated in 9bfef93